pkg/operations/kubernetesupgrade/upgrader.go

```diff
@@ -497,6 +505,11 @@ func (ku *Upgrader) upgradeAgentScaleSets(ctx context.Context) error {
 
 			ku.logger.Infof("Successfully set capacity for VMSS %s", vmssToUpgrade.Name)
 
+			cordonDrainTimeout := defaultCordonDrainTimeout
```

Comment:
Let's settle on a common "use default value if not provided" pattern. Lines 269-273 are doing the same thing, it seems, but in a sort of inverse way.

Reply:
Done